omero/sysadmins/server-advanced-configuration.rst

```diff
@@ -129,18 +129,18 @@ To run multiple servers on a single host, the easiest approach is to prefix all
 ports (SSL, TCP, registry) using :property:`omero.ports.prefix`::
 
     # First server
-    cd ~/OMERO.server-1
-    bin/omero admin start
+    OMERODIR = ~/OMERO.server-1
```

Comment:
```suggestion
    export OMERODIR=~/OMERO.server-1
```
You can't have spaces around the `=` when defining a shell environment variable. I think `export` is required (I haven't tested this, we should schedule a full test of all the docs before the RC release)

Reply:
space added by mistake